runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -83,6 +83,11 @@ public GroundItemsOverlay(Client client, GroundItemsPlugin plugin, GroundItemsCo
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		if (plugin.isHideAll())
```

Comment:
include the plugin.isHotkeyPressed check here too, if it is true, ignore hiding of all items

Reply:
Good point, woopsie